src/pybind/mgr/cephadm/templates/services/haproxy/haproxy.cfg.j2

```diff
@@ -0,0 +1,47 @@
+# {{ cephadm_managed }}
+global
+    log         127.0.0.1 local2
+    chroot      /var/lib/haproxy
```

Comment:
Maybe we should change this...

Adam made the changes to use  a specific "datadir" for HAProxy , we are creating this "dir" and  we will use it for store this config file once rendered.
what if we put the chroot folder and store the pidfile in the "datadir" folder?
it will be like:

```
chroot {{ haproxy_datadir }}
pidfile {{ haproxy_datadir }}/haproxy.pid
```

Reply:
I thought the directory after "chroot" was the directory within the container? While, on the host machine, we are storing the config file in a sub-directory of the haproxy data dir that directory is being mounted to /var/lib/haproxy within the container. As far as the haproxy container is concerned the config file is located in /var/lib/haproxy. Unless the directory listed here is actually referring to one on the host machine instead of in the container (which seems odd) I don't think it makes sense to change it. I think it would make sense, however,  to move the pidifle into /var/lib/haproxy so that you can see it from the host machine (`pidfile /var/lib/haproxy/haproxy.pid`)

Reply:
You have convinced me.